terraform/variables.tf

```diff
@@ -28,6 +28,26 @@ variable "data_snapshot_s3_bucket" {
   default     = "univaf-data-snapshots"
 }
 
+variable "db_user" {
+  description = "The database user"
+  default     = "univaf"
+}
+
+variable "db_password" {
+  description = "The password for the database instance, filled via Terraform"
+  sensitive   = true
+}
+
+variable "db_instance" {
+  description = "The instance type for the DB. Reference: https://aws.amazon.com/rds/instance-types/"
+  default     = "db.t4g.small"
```

Comment:
Was previously `db.m5.large` (2 CPU / 8 GB RAM). This is 1 CPU (ARM) / 2 GB RAM, and matches what we have on Render (which seems to be fine). Costs ~$25/mo instead of ~$135/mo.

Reply:
This is only the second time I’ve used Graviton instances for anything! They’re a little cheaper, and *should* perform just as well on less power. We can bump this up pretty easily if we see problems.